### PR TITLE
DOCSP-38510 create python language landing page

### PR DESCRIPTION
**Pull request by mmaville-mdb**
Setting up structure for project and adding the Python landing page.

**For Reviewer/SME**: 

- We still need to finalize if and how we're going to nest the community links. I added the community link tags that should be available to render during publishing on 4/29. Any guidance on formatting the sub-heading or indentation in this section is appreciated.

- Beanie and ODMantic look like separate projects, unless there is a specific page we're providing guidance to for both.

- All links were added by me and should be verified by an SME. We may want to direct them to the documentation over a landing page, or a github project over the documentation, etc.

Staging link: https://preview-mongodbmmavillemdb.gatsbyjs.io/landing/DOCSP-38510-Create-Python-Language-Landing-page/languages/python-language/

Build log: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=662ac599b67dbf804a5241cc

### CODE REVIEW

File: source/languages/python-language.txt [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+.. community-driver:: MongoEngine for Flask
+   :url: https://docs.mongoengine.org/projects/flask-mongoengine/en/stable/
+
+.. community-driver:: Beanie for FastAPI
+   :url: https://beanie-odm.dev/
+
+.. community-driver:: ODMantic for FastAPI
+   :url: https://art049.github.io/odmantic/
+
+Libraries and Extensions
```
Comment (mmaville-mdb) [inline source/languages/python-language.txt:78]: Padding Issue seems to be occuring here.

File: source/languages/python-language.txt [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+.. card-group::
+   :columns: 2
+
+   .. card::
+      :headline: PyMongo
+      :url: https://www.mongodb.com/docs/drivers/pymongo/
+      :icon: /images/icons/python.svg
+      :icon-alt: Driver card icon
+
+      Provides a synchonus API for working with MongoDB databases consistent with other MongoDB language drivers.
```
Comment (mongoKart) [inline source/languages/python-language.txt:36]:
```suggestion
      Provides a synchronous API for working with MongoDB databases consistent with other MongoDB language drivers.
```

File: source/languages/python-language.txt [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+   .. card::
+      :headline: Motor
+      :url: https://www.mongodb.com/docs/drivers/motor/
+      :icon: /images/icons/python.svg
+      :icon-alt: Driver card icon
+
+      Provides an asynchonus API for working with MongoDB databases consistent with other MongoDB language drivers.
```
Comment (mongoKart) [inline source/languages/python-language.txt:44]:
```suggestion
      Provides an asynchronous API for working with MongoDB databases consistent with other MongoDB language drivers.
```

File: source/languages/python-language.txt [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      :icon: /images/icons/Microsoft.png
+      :icon-alt: Microsoft logo
+
+   .. card::
+      :headline: HayStack
+      :url: https://github.com/deepset-ai/haystack
+      :icon: /images/icons/Haystack-1.png
+      :icon-alt: Haystack logo
+
+More ways to learn
```
Comment (mongoKart) [inline source/languages/python-language.txt:114]:
```suggestion
More Ways to Learn
```

**Review by mongoKart — COMMENTED** [review]
couple small things

File: source/languages/python-language.txt [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+   .. card::
+      :headline: View Python developer resources
+      :url: https://www.mongodb.com/developer/languages/python/
+      :icon: /images/technical-mdb-devhub.svg
+      :icon-alt: DevHub icon
+
+      Python Resources  →
+
+   .. card::
+      :headline: Take the Getting Started with Python course
```
Comment (caitlindavey) [inline source/languages/python-language.txt:130]:
```suggestion
      :headline: Learn about Python and MongoDB 
```

**Pull request merged**